bottlenecks.md

```diff
@@ -11,189 +11,255 @@ kernelspec:
 
-The SFS is convenient analytically, because it only depends on the mean length and frequency of genealogical branches. For many demographic models of interest the means can be derived analytically either using coalescent theory (cite Huang, TPB) or diffusion equations (cite dadi). A number of composite likelihood approaches have been developed based on either analytic results for the SFS (cite dadi Excoffier, Jaada). However, analytic expectations for the SFS break down for large samples and/or complex demographic models. 
+Here we simulates a single sample of n=12 from a population that underwent an instantaneous bottleneck 0.4 * 2N generations ago. Note that since msprime is assuming a diploid population we set the initial population size to 1/2 to obtain coalescence times scaled in units of $2N_e$ generations:
 
-In the following section we show how the SFS can be approximated using coalescence simulations and compare such approximations to analytic results. We will assume a simple toy history of a single panmictic population that is affected by an instaneous bottleneck at time T with strenght s (cite Galtier et al). The effect of this bottleneck is to induce sudden burst of coalescence, which simultaneous multiple merges. We measure bottleneck strength as the probability that a pair of lineages coalesces during the bottleneck (we could could of course convert s into on (imaginary) time period that would give the same probability of coalescence $s=1-e^{-T}$).
-
-We assume a sample of 10 haploid genomes and use msprime to simulate 10,000 replicate genealogies. For each genealogy the function approx_SFS records the unfolded SFS as the mean length of branches with n leafnodes (normalized by the total length of the genealogy) by iterating through all nodes in the tree.sequence. Note that we are simulating genealogies only, i.e. we do not need to simulate mutations.
-
-We use a for loop to record the SFS for a range of bottleneck strengths parameters in a dictionary:
 
```

Comment:
This will make things run on time for now:
```suggestion
  ```{code-cell} ipython3
  :"tags": ["remove_input"]
  # REMOVE WHEN TSKIT 0.3.7 RELEASED
  import tskit
  import functools
  tskit.metadata.parse_metadata_schema = functools.lru_cache(maxsize=128,(tskit.metadata.parse_metadata_schema)
    ```
```